packages/11ty/_lib/uid/index.js

```diff
@@ -0,0 +1,38 @@
+function* UidGenerator() {
+  let i = 0
+  while (true) yield i++
+}
```

Comment:
@hbalenda though a simply incrementer will work under most conditions it is not guaranteed to be unique. We might instead emplement a `uid` generator using `Math.random()`, `Date.now()`, a combination of the two, or a `Uuid` library.

Reply:
For reference, [Uuid](https://github.com/uuidjs/uuid) is widely used, however [ShortUniqueId](https://github.com/simplyhexagonal/short-unique-id) is perhaps better suited to this requirement.